mon-pix/app/styles/pages/_competence-results.scss

```
@@ -4,44 +4,64 @@
     flex-direction: column;
     align-items: center;
     justify-content: center;
-    height: 360px;
     margin: 4px;
     border-radius: 5px;
 
+    &-result-comment {
+      margin: 0;
+      color: $white;
+      font-size: 3rem;
+      font-family: $font-open-sans;
+      font-weight: 700;
+    }
+
     @include device-is('tablet') {
-      height: 240px;
+      padding: 30px;
       margin: 12px;
     }
 
     &--congrats {
-      background: url('/images/banners/congrats.svg');
-      justify-content: flex-end;
+      background: url(/images/banners/congrats-mobile.png) center -240px no-repeat, linear-gradient(0deg, rgb(18, 163, 255) 0%, rgb(61, 104, 255) 100%);
```

```suggestion
      background: url('/images/banners/congrats-mobile.png') center -240px no-repeat, linear-gradient(0deg, rgb(18, 163, 255) 0%, rgb(61, 104, 255) 100%);
```
Il manque les `'` autour de l'URL.